changelog.md

```diff
@@ -11,12 +11,17 @@ Please look at the following changes to check whether you have to verify your se
 
 ##### Restrict the number of concurrent HTTP requests
 
-With this number of Marathon it is possible to restrict the number of concurrent http requests.
-This prevents Denial of Service behavior, that has been reported in some installations.
-If the limit of concurrent requests is reached, a HTTP 503 Service Temporarily Unavailable is returned,
+With this version of Marathon it is possible to restrict the number of concurrent http requests,
+that are handled concurrently in the service layer.
+This enables the Marathon service to back pressure too much requests and prevents Denial of Service 
+behavior, that has been reported in some installations.
```

Comment:
IMO the following wording would be a bit clearer: This enables the Marathon service to apply back pressure when receiving too many requests and helps prevent Denial of Service attacks.